docs/docs/cmd/spo/site/site-classic-list.md

```diff
@@ -53,4 +56,10 @@ List all classic project sites that contain _project_ in the URL
 
 ```sh
 m365 spo site classic list --webTemplate PROJECTSITE#0 --filter "Url -like 'project'"
-```
\ No newline at end of file
+```
+
+List all OneDrive sites
```

Comment:
Let's rephrase this to `List only OneDrive sites` to disambiguate from the other sample that we have above and which includes both OneDrive and regular sites.